terraform/main.tf

```diff
@@ -0,0 +1,12 @@
+resource "juju_application" "opensearch" {
+  charm {
+    name     = "opensearch"
+    channel  = var.channel
+    revision = var.revision
+  }
```

Comment:
**todo:** As far as I can see, you aren't using `base` here. Revision is *probably* tighter, but was this an omission?

Reply:
Yes, it was  :) thanks!